src/plugins/data_table/settings_manager.js

```diff
@@ -0,0 +1,79 @@
+var SettingsManager = Backdraft.Utils.Class.extend({
```

Comment:
I like having all user preference related code in one place, like the SettingsManager. However, this may become unnecessary if this behavior is integrated into ActionTable. The hooks into the events  ```table.on("draw"... ```, ```table._columnManager.on("etc"... ``` can be created inside of the ActionTable plugin.